src/pal/src/init/pal.cpp

```diff
@@ -680,8 +680,17 @@ PAL_IsDebuggerPresent()
     }
 
     return debugger_present;
+#elif defined(__APPLE__)
```

Comment:
It would be nice to use HAVE_SYSCTL here to make it just work for FreeBSD, and HAVE_PROCFS_CTL instead of Linux to make it symmetric.